app/views/users/bounties/index.html.erb

```diff
@@ -6,3 +6,5 @@
 <div id="bounties" class="min-w-full">
   <%= render @bounties %>
 </div>
+
+<%== pagy_nav(@pagy).html_safe %>
```

Comment:
Feel free to ignore, but i'll throw in my two cents:

* `html_safe` is redundnat: see here: https://edgeguides.rubyonrails.org/active_support_core_extensions.html#output-safety